lib/ansible/modules/cloud/azure/azure_rm_storageaccount_facts.py

```diff
@@ -110,16 +293,21 @@ def __init__(self):
             name=dict(type='str'),
             resource_group=dict(type='str', aliases=['resource_group_name']),
             tags=dict(type='list'),
+            show_connection_string=dict(type='bool'),
+            show_blob_cors=dict(type='bool')
```

Comment:
is the show_blob_cors necessary? why return it by default? we add flag  for show credentials like connection_string for security concern, although weak. but is cors info something need special flag?

Reply:
My thought is getting the blob cors setting needs another API call (coming table/queue/file may need one API call for each), which will cause a lot of network overhead when listing resources in a resource group or subscription. So I make this as an option to let user explicitly get the CORS setting.